docs/File-Formats.md

```diff
@@ -1105,58 +1105,44 @@ rank<TAB>gene<TAB>N<TAB>n<TAB>p<TAB>q
-2. **description**: A descripion of the gene panel.
-3. **gene_list**: Tab separated genes, represented either by all hugo symbols or all entrez_gene_ids.
+To include gene panel data in your instance, the following data and/or configurations can be used:
+1. **Gene panel file**: This file contains the genes on the gene panel. A panel can be used for multiple studies within the instance and should be loaded prior to loading a study with gene panel data. For information on the format and import process please visit: [Import-Gene-Panels](Import-Gene-Panels.md).
+2. **Gene panel matrix file**: This file is used to specify which samples are sequenced on which gene panel in which genetic profile. This required for mutation and fusion data, because the MAF and fusion formats are unable to include samples which are sequenced but no mutations are found. For other genetic data, columns can be added to specify their gene panel, but a property can also be added to their respective meta file, because these data files contain all profiled samples. Although the gene panel matrix functionality overlaps with the case list functionality, a case list for mutations (`_sequenced`) is also required.
+3. **Gene panel property in meta file**: Adding the `gene_panel:` property to the meta file of a non-mutation data file, will assign all samples from that data type to the gene panel. In this case it is not necessary to include a column for this profile in the gene panel matrix file.
 
-An example gene panel file would be:
-```
-stable_id: IMPACT410
-description: Targeted (410 cancer genes) sequencing of various tumor types via MSK-IMPACT on Illumina HiSeq sequencers.
-gene_list: ABL1    ACVR1   AKT1    AKT3 ...
-```
-
-For information on importing gene panels please visit: [Import-Gene-Panels](Import-Gene-Panels.md).
-
-#### Sample-Profile Matrix
-
-The second component to gene panel data is associating samples and profile to the panel which applies. The following column is required :
+#### Gene Panel Matrix file
+The gene panel matrix file contains a list of samples in the first column, and an additional column for each profile in the study using the stable_id as the column header. These stable_id's should match the ones in their respective meta files. Fusion events are saved in the mutation table in the cBioPortal database, so they should be included in the `mutations` column. Other than mutation and fusion data formats can use the `gene_panel:` meta property if all samples are profiled on the same gene panel.
 
-- ***SAMPLE_ID***: Sample Id from the study 
+For each sample-profile combination, a gene panel should be specified by using the stable_id from the gene panel file, for example `mutations` for mutation data and `gistic` for discrete CNA data, or `NA` to reflect whole exome. Columns should be separated by tabs.
```

Comment:
Suggest WES/WGS instead of NA.

Reply:
Done

Reply:
What I see currently in the PR is a choice between three options: "WES", "WGS", or "NA". I think what @n1zea144 wanted was to avoid "NA" which would mean something like "we don't know". I also think his request intended that there be a single unique string which represents "either WES or WGS", rather than giving the user an option to distinguish between WES and WGS. So I interpret the request as forbidding "NA", and requiring either a proper gene panel name, or the string "WES/WGS" ... but maybe @n1zea144 can clarify.

Reply:
HI @sandertan, sorry for the delay, maybe @sheridancbio let you know I was out of the lab last week.  In any event, I was thinking what Rob said.  Currently, when no gene panel is specified, I believe the frontend (oncoprint) assumes WGS.  Why allow NA when most likely WGS is likely or would result in proper behavior in the frontend.

I was thinking that we distinguish between WGS and WES in the matrix file, but admittedly I'm not sure if we would make much of a distinction between WES or WGS in terms of behavior in the portal.  I do imagine that this information is at hand to a study curator, so why not support  both independently?  A change like this should be coordinated with the frontend.  I would bring @adamabeshouse into the conversation.

Reply:
whatever you choose, we'll be able to handle on the frontend. it just depends on what we want to show, which I would defer to @jjgao for

Reply:
This would require a database change because this info can’t be saved in the gene panel column of the sample_profile table. Perhaps we can move this as an enhancement to a separate github issue?

As mentioned below in separate comment, I restored NA as only possible value. Adding functionality for WES/WGS is I think out of scope for this PR.

I'll squash & rebase if you think it's okay in its current state.

Reply:
@sandertan I don't know the use case for introducing NA into the matrix file, but if we are going to allow it, I think we should hear from @jjgao as to what changes need to happen on the frontend so they can be coordinated with this PR.